experimental/packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -458,6 +478,31 @@ describe('HttpInstrumentation', () => {
         });
       }
 
+      it('should not trace ignored requests with headers (client and server side)', async () => {
```

Comment:
Is there a test somewhere for the opposite case - where the hook returns `false` and the trace should not be ignored?

Reply:
Updated :)